experiment/measurer/measure_manager.py

```diff
@@ -755,51 +700,294 @@ def get_pool_args(measurers_cpus, runners_cpus):
     return (measurers_cpus, _process_init, (cores_queue,))
 
 
-def measure_manager_loop(experiment: str,
-                         max_total_time: int,
-                         measurers_cpus=None,
-                         region_coverage=False):  # pylint: disable=too-many-locals
-    """Measure manager loop. Creates request and response queues, request
-    measurements tasks from workers, retrieve measurement results from response
-    queue and writes measured snapshots in database."""
-    logger.info('Starting measure manager loop.')
-    if not measurers_cpus:
-        measurers_cpus = multiprocessing.cpu_count()
-        logger.info('Number of measurer CPUs not passed as argument. using %d',
-                    measurers_cpus)
-    with multiprocessing.Pool() as pool, multiprocessing.Manager() as manager:
-        logger.info('Setting up coverage binaries')
-        set_up_coverage_binaries(pool, experiment)
-        request_queue = manager.Queue()
-        response_queue = manager.Queue()
-
-        config = {
-            'request_queue': request_queue,
-            'response_queue': response_queue,
-            'region_coverage': region_coverage,
-        }
-        local_measure_worker = measure_worker.LocalMeasureWorker(config)
-
-        # Since each worker is going to be in an infinite loop, we dont need
-        # result return. Workers' life scope will end automatically when there
-        # are no more snapshots left to measure.
-        logger.info('Starting measure worker loop for %d workers',
-                    measurers_cpus)
-        for _ in range(measurers_cpus):
-            _result = pool.apply_async(local_measure_worker.measure_worker_loop)
+class BaseMeasureManager:
+    """Base class for measure manager. Encapsulates core methods that will be
+    implemented for Local and Google Cloud measure managers."""
+
+    def __init__(self, experiment: str, region_coverage=False):
```

Comment:
nit: Specify the type of `region_coverage` for consistency, given that `experiment` has type hinting.